content/pages/2024/tickets.md

```diff
@@ -0,0 +1,53 @@
+---
+title: EuroSciPy 2024 - Tickets
+url: 2024/tickets.html
+save_as: 2024/tickets.html
+section: euroscipy_2024
+template: page
+slug: tickets_2024
+---
+
+# Tickets for EuroSciPy
```

Comment:
Optionally, we could say that the opening of the ticket will start after the review process and the keynot are announced.